app/models/rent.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Rent < ApplicationRecord
+  validates :user, :book, :from, :to, presence: true
```

Comment:
What do you think about checking that the dates are valid within date bounds? Meaning that the `from` date comes before the `to` date.

Reply:
it's a good idea!!